plexpy/notifiers.py

```diff
@@ -2754,32 +2724,82 @@ class HIPCHAT(object):
     def __init__(self):
         self.apiurl = plexpy.CONFIG.HIPCHAT_URL
         self.color = plexpy.CONFIG.HIPCHAT_COLOR
-        self.incl_subject = plexpy.CONFIG.HIPCHAT_INCL_SUBJECT
         self.emoticon = plexpy.CONFIG.HIPCHAT_EMOTICON
+        self.from_subject = plexpy.CONFIG.HIPCHAT_FROM_SUBJECT
+        self.incl_pmslink = plexpy.CONFIG.FACEBOOK_INCL_PMSLINK
+        self.incl_poster = plexpy.CONFIG.FACEBOOK_INCL_POSTER
+        self.incl_subject = plexpy.CONFIG.HIPCHAT_INCL_SUBJECT
 
-    def notify(self, message, subject):
+    def notify(self, message, subject, **kwargs):
         if not message or not subject:
             return
-        
+
+        data = {'notify': 'false'}
+
+        text = message.encode('utf-8')
+
         if self.incl_subject:
-            text = subject.encode('utf-8') + ': ' + message.encode('utf-8')
-        else:
-            text = message.encode('utf-8')
+            text = subject.encode('utf-8') + ': ' + text
 
         if self.emoticon:
```

Comment:
Can't have emoticon if "Include Poster" is checked.

The notification still appends `(plexpy)` to the beginning.